docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,231 @@
+
+**Documentation PRs:** If working specifically on markdown files or other documentation, be sure to check the actual look of the document once you have pushed your PR and fix any formatting errors. You can do this by going to “Files Changed” and viewing how your markdown file will actually appear in GitHub. Don’t forget to click all links to make sure they won't be broken when the document is rendered. 
+
+## 4. Code Review
+
+### Testing or Contributing to someone's PR
+First, make a remote of their fork:
+
+    git remote add "their_fork" <their_fork_url>
+          
+Then, fetch their remote fork and checkout the branch related to their PR:
+
+    git fetch their_fork
+    git checkout their_fork/branch_name
+
+Alternatively, to avoid being under the detached HEAD state, checkout and create a local branch with the command
+
+    git fetch their_fork
+    git checkout -b branch their_fork/branch_name
+
+Follow the provided testing instructions to test their code on your VM. 
+
+If you made changes (with their permission), you can commit and push them to that branch. Note that you will most likely need to be given permission on GitHub by 
+the developer to push changes to their branch.
+
+    git commit -m "Changes made"
+    git push their_fork branch_name
+
+Don’t forget to consult the Links (see above) for coding review standards and guidelines and the code review checklist.
+
+### Labels
+The `Passed Manual Tests` label can be added to a PR if a reviewer has tested the code on their own VM and determined that it performs as intended.
+
+The `Needs Rebase` label can be added if a PR has merge conflicts with the branch it is based off of.
+
+The `Needs Work` label can be added if a PR is not yet ready for review and requires more work before it can be approved.
+
+Other Labels can also be added by reviewers at their discretion. 
+
+## 5. Rebasing
+Rebasing is used when there are conflicts between your branch and the branch your PR is based off of (an alternative to merging). 
+Here is a general workflow:
+
+    cd /var/www/loris
+    git checkout branch_name
+    git fetch aces
+    git rebase aces/main
+    (#) while (conflicts)  { 
+        // fix any conflicts and add any files
+        git add  filename
+        git rebase --continue
+        // OR you can "skip" specific changes
+        git rebase --skip
+    # } loop back to (#) - fix any remaining conflicts.... until no more conflicts
+    git push --force-with-lease my_fork branch_name
+
+### Fixing a rebase gone wrong
+1. The most basic way of “fixing” a rebase gone wrong is to go back to the commit right before the rebase. The easiest way to do this is to either reset or revert to a previous commit. **Note** that this is a potentially destructive command, so be very cautious when using it. It is also recommended that you make a backup of your branch before performing these commands so that you don't lose any changes. 
+For example:
+
```

Comment:
Remove this line break, otherwise RTD will start another numbered list (resulting in two consecutive 1.)